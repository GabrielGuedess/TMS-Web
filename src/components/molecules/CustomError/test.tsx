import { render } from '@testing-library/react';

import { CustomError } from '.';

jest.mock('lottie-react', () => ({
  __esModule: true,
  default: function Mock() {
    return <div data-testid="Mock Lottie" />;
  },
}));

describe('<CustomError />', () => {
  it('should render the custom error', () => {
    const { container } = render(
      <CustomError noRowsMessageFunc={() => 'Error'} />,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
