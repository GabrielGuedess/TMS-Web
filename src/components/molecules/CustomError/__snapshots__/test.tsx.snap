// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CustomError /> should render the custom error 1`] = `
<div
  class="flex h-[38.563rem] w-full flex-col items-center justify-center md:h-[35.5rem]"
>
  <div
    data-testid="Mock Lottie"
  />
  <span
    class="text-base text-comet-500 transition-all dark:text-dark-300"
  >
    Error
  </span>
</div>
`;
